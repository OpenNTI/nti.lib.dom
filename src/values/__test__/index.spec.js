/** @jest-environment node */
/* eslint-env jest */
import * as nodes from '../index';

test ('index', () => {
	expect(Object.keys(nodes)).toMatchSnapshot();
});
