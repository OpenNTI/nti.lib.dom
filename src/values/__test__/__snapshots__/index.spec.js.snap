// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index 1`] = `
Array [
  "enforceNumber",
  "inRange",
  "isValueEmpty",
  "retargetAnchorsWithExternalRefs",
  "sanitizeExternalContentForInput",
]
`;
